hal/src/b5som/network/quectel_ncp_client.cpp

```diff
@@ -152,51 +152,111 @@ int QuectelNcpClient::initParser(Stream* stream) {
     auto parserConf = AtParserConfig().stream(stream).commandTerminator(AtCommandTerminator::CRLF);
     parser_.destroy();
     CHECK(parser_.init(std::move(parserConf)));
+    //+CREG: <n>,<stat>[,<lac>,<ci>[,<Act>]]
```

Comment:
would a comment here about the difference between lines 155 and 156 be useful for readers who do not know anything about AT commands?
For example, commenting that first line is for Query, and second line is a URC.

Reply:
Done